mne/export/_edf_bdf.py

```diff
@@ -230,11 +258,24 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
         else:
             annotations.append(EdfAnnotation(onset, duration, desc))
 
-    Edf(
+    annotations.extend(pad_annotations)
```

Comment:
IIUC, doing it this way will change the order of the items in the `annotations` list (previously, all the BAD_ACQ_SKIP` would be at the beginning, now they will be at the end).  Does that matter?  Or does EDF/BDF not care (or does the writer sort the annots in any case)?